pkg/cmd/server/admin/create_nodeconfig.go

```diff
@@ -155,8 +155,14 @@ func (o CreateNodeConfigOptions) Validate(args []string) error {
 	if len(o.APIServerURL) == 0 {
 		return errors.New("--master must be provided")
 	}
-	if _, err := os.Stat(o.APIServerCAFile); len(o.APIServerCAFile) == 0 || err != nil {
-		return fmt.Errorf("--certificate-authority, %q must be a valid certificate file", cmdutil.GetDisplayFilename(o.APIServerCAFile))
+	if len(o.APIServerCAFiles) == 0 {
+		return fmt.Errorf("--certificate-authority must be a valid certificate file")
+	} else {
+		for _, caFile := range o.APIServerCAFiles {
+			if _, err := os.Stat(caFile); len(caFile) == 0 || err != nil {
```

Comment:
Do an actual parse?  I just noticed you did it in the `create_mastercerts.go` file.